packages/stylelint-plugin/package.json

```diff
@@ -28,11 +28,11 @@
     "merge": "^1.2.1",
     "stylelint-config-prettier": "^8.0.1",
     "stylelint-order": "^4.0.0",
-    "stylelint-prettier": "^1.1.2",
-    "stylelint-scss": "^3.16.0"
+    "stylelint-prettier": "^1.2.0",
+    "stylelint-scss": "^3.19.0"
   },
   "devDependencies": {
-    "stylelint": "^13.2.1"
+    "stylelint": "^13.12.0"
```

Comment:
Are there any [new rules](https://github.com/stylelint/stylelint/blob/master/CHANGELOG.md) in the new releases that we might be interested in adding into our config?

Same with the [stylelint-scss changes](https://github.com/kristerkari/stylelint-scss/blob/master/CHANGELOG.md).

Reply:
https://github.com/stylelint/stylelint/blob/master/CHANGELOG.md

> Added: ignoreDisables, reportNeedlessDisables, reportInvalidScopeDisables and reportDescriptionlessDisables configuration object properties

That sounds really handy. Previously eslint-"no uneeded comments"- sorta stuff had to be configured as arguments. being able to put that into shared config is super nice